src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -73,7 +74,85 @@ protected virtual IEnumerable<FieldType> ProvideFields()
         {
             var typeInformation = GetTypeInformation(memberInfo);
             var graphType = typeInformation.ConstructGraphType();
-            return AutoRegisteringHelper.CreateField(memberInfo, graphType, false);
+            var fieldType = AutoRegisteringHelper.CreateField(memberInfo, graphType, false);
+            if (memberInfo is MethodInfo methodInfo)
```

Comment:
```suggestion
            if (memberInfo is MethodInfo methodInfo && methodInfo.GetParameters().Length == 0)
```
\+ delete
```
 if (methodInfo.GetParameters().Length == 0)
            {
                return (null, MemberResolver.Create(methodInfo));
            }
```
from  `BuildMethodResolver`. Or keep as is since ` BuildMethodResolver` is protected and may be called by somebody else.

Reply:
> Or keep as is since BuildMethodResolver is protected and may be called by somebody else.

I would like to leave as-is.